src/System.Management.Automation/engine/CommandDiscovery.cs

```diff
@@ -1281,27 +1281,6 @@ internal void UnregisterLookupCommandInfoAction(string currentAction, string com
         private HashSet<string> _activeCommandNotFound = new HashSet<string>(StringComparer.OrdinalIgnoreCase);
```

Comment:
@iSazonov, your last commit had 2 failures in `PowerShell-CI-linux`
Command Discovery tests.Native command discovery.Can discover a native command without extension
```powershell
Expected regular expression 'ping' to match $null, but it did not match.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/engine/Basic/CommandDiscovery.Tests.ps1: line 223
223:             (Get-Command -Name "Ping" -CommandType Application).Name | Should -Match $expectedName
```
Command Discovery tests.Native command discovery.Error occurred in Context block
```powershell
Cannot process argument transformation on parameter 'Skip'. Cannot convert value "System.String" to type "System.Management.Automation.SwitchParameter". Boolean parameters accept only Boolean values and numbers, such as $True, $False, 1 or 0.
At /home/vsts/work/1/s/test/powershell/engine/Basic/CommandDiscovery.Tests.ps1:226 char:76
```